mesonbuild/dependencies/platform.py

```diff
@@ -17,24 +17,43 @@
 
 from .. import mesonlib
 
-from .base import ExternalDependency, DependencyException
+from .base import ExtraFrameworkDependency, ExternalDependency, DependencyException
 
 
 class AppleFrameworks(ExternalDependency):
     def __init__(self, env, kwargs):
         super().__init__('appleframeworks', env, None, kwargs)
         modules = kwargs.get('modules', [])
+        includes = kwargs.get('include', [])
+
         if isinstance(modules, str):
             modules = [modules]
         if not modules:
             raise DependencyException("AppleFrameworks dependency requires at least one module.")
-        self.frameworks = modules
-        # FIXME: Use self.compiler to check if the frameworks are available
+
+        if isinstance(includes, str):
+            includes = [include]
+        if includes:
+            includes = ['/System/Library/Frameworks', '/Library/Frameworks'] + includes
+
+        self.frameworks = []
+
+        for f in modules:
+            self.frameworks += [ExtraFrameworkDependency(
+                f,
+                True,
+                includes,
+                env,
+                None,
+                kwargs,
+                )]
```

Comment:
__[Flake8]__

[E123] closing bracket does not match indentation of opening bracket's line


<!-- Comment by [SideCI](https://sideci.com) -->